### PR TITLE
Testing Yarn

### PR DESCRIPTION
**Pull request by mute**
## PR Checklist
- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [x] Edge (Win 10)
- ~~[ ] Unit tests written (`common` at minimum)~~
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- ~~[ ] One core team UX approval~~

### CODE REVIEW

**Comment by jondlm** [comment]
Dang, that's *much* faster. I'm gonna bump this branch to node 6 as well to see if that helps even more.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "preversion": "rm -rf dist && gulp lint && gulp preversion && npm run build-bundle",
     "postversion": "gulp docs-upload",
     "start": "npm run dev",
-    "test": "gulp lint && npm run test-browserstack && npm run test-acceptance",
+    "test": "gulp lint && npm run test-browserstack",
```
Comment (sodiumjoe) [inline package.json]: http://i1.kym-cdn.com/photos/images/original/000/755/601/c2a.gif

**Comment by mute** [comment]
We should probably not merge this yet -- I just opened the PR to facilitate some performance testing.

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,16 @@
-before_install: bash ./scripts/validate-tests.sh
-after_success: bash ./scripts/docs-upload-gh.sh
+before_install:
+  # Repo for Yarn
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
```
Comment (jondlm) [inline .travis.yml]: I wonder why `npm i yarn` doesn't work. Seem like some weird hoops to jump through, I'm fine with it if that's what's needed though.

File: .travis.yml [hunk 3]
```diff
@@ -9,5 +20,7 @@ env:
     - secure: "Recue3+sPDb4kQtKftZuYa0zwg6/+H+QO5tJP8WGa+Xq7OZBGzAt/bADW0Q7KvUPxtK6qUCynuWrnaBlEWc+UKIHlrcHCpFPn0zkpzGvzX6a+tvhjHAQNDB7x/VYORvYEQqMzWC5t+qLj6C9ffjXP+6uCo+o+y/5K5ydEyqa2e42LQblSJmiiCG754CqV1LxmasklGAC90+ldR6d7PFFpqXDc/BIsyDAIuS4FXfjyli+VnvCza7PQJ4XGkCoywmBNg8bcR82UwduA854/Gr7oYKZRUamMpBuQY25EupjwZ/GV2OyDSK3UcO3blEVrjRnP3gC9Ljd+WvcBqy7d1FJTCk0Vsy/pstxjVpP19H3I6xOU41buB3Pw1aLTkIp+PFmhoab5fu03bWfwUp/N1Xl1URKWcIKokwh/98NSA8OZ16VbMhFkxuUCs5bsFR+DB6SLdc2XNpzzdxdoe3XZHB5LxVOIoJY6CiskfYdNBPsEo+NZTrv+eUk+l4gMNTIYwgCWEbbIvXE6CwFITg9ERQhY7fvGJ37DOWfvLfxKnDc0jFqnlUnWgrtDppx7akE1vgfFu9W8jWRF4phGU/O4BwWUNhLtoGUgTH6RR4+4xoNv3LB1g6fnd50jUin4f6o2KSOkdPZFPmv74MygC7WyPERWR2IjpT2x+yfPIa6TetlY/c="
 language: node_js
 node_js:
-  - "4.1"
-
+  - "6"
```
Comment (jondlm) [inline .travis.yml]: Cool beans

File: scripts/env-check.sh [hunk 4]
```diff
@@ -1,9 +0,0 @@
-if [ -z $BROWSERSTACK_USERNAME ]; then
```
Comment (jondlm) [inline scripts/env-check.sh]: Is this file being used anywhere else?

File: src/components/DropMenu/DropMenu.spec.jsx [hunk 5]
```diff
@@ -805,12 +805,12 @@ describe('DropMenu', () => {
 
 				const flyOutDOMNode = document.querySelector('.lucid-DropMenu.lucid-ContextMenu-FlyOut .lucid-DropMenu-option-container');
 				assert.equal(flyOutDOMNode.children.length, 9);
-				assert.equal(flyOutDOMNode.children[0].innerText, 'Preferred');
+				assert.equal(flyOutDOMNode.children[0].textContent, 'Preferred');
```
Comment (jondlm) [inline src/components/DropMenu/DropMenu.spec.jsx]: 👍

File: .travis.yml (hunk 2 above)
Comment (mute) [inline .travis.yml]: `npm i yarn` should actually work -- this is just the recommended path to getting it installed per the Yarn docs: https://yarnpkg.com/en/docs/install-ci#travis-tab

I think it's probably just a matter of the speed of getting the binary onto the box.

File: scripts/env-check.sh (hunk 4 above)
Comment (mute) [inline scripts/env-check.sh]: I suppose it is -- I'll delete more things.

Comment (sodiumjoe) [inline scripts/env-check.sh]: only in npm scripts, can go away when the browserstack stuff goes away

**Pull request merged**